webapp/src/ts/modules/messages/messages.routes.ts

```diff
@@ -31,6 +31,7 @@ export const routes: Routes = [
               }
             };
           }
+          return null;
```

Comment:
This change was required because of how this code needs to implement `Route` and that specifies the return values for `Route.matcher` (which includes `null` but not `undefined`)